.theme/package.json

```diff
@@ -41,10 +41,10 @@
     "@rollup/rollup-linux-x64-gnu": "^4.34.6"
   },
   "dependencies": {
-    "@primevue/forms": "^4.2.5",
-    "@primevue/themes": "^4.2.5",
+    "@primevue/forms": "^4.3.2",
+    "@primevue/themes": "^4.3.2",
     "async-mutex": "^0.5.0",
     "primeicons": "^7.0.0",
-    "primevue": "^4.2.5"
```

Comment:
there was a problem with primevue minor upgrades, make them `~4.2.5` to avoid unintentional `4.3.x` upgrades in package-lock.json